model/scope.go

```diff
@@ -113,19 +113,19 @@ const (
 	ScopeUsersUpdate Scope = "users:update"
 )
 
+var searchAndCRUD = []ScopeOperation{ScopeCreate, ScopeRead, ScopeUpdate, ScopeDelete, ScopeSearch}
```

Comment:
Maybe rename to `searchAndCRUDScopes` since this is in package namespace, even if it is private?